p2panda-rs/src/document/document_id.rs

```diff
@@ -76,23 +83,6 @@ impl FromStr for DocumentId {
     }
 }
 
-impl<'de> Deserialize<'de> for DocumentId {
```

Comment:
Oh nice, this works because deriving Deserialize calls our implementation on Hash which validates everything (hex encoding, length, multihash format), right?

Reply:
Yeh, exactly! It's very neat :+1: